### ISSUE TITLE
questions about anomaly labels in ue.csv

### ISSUE THREAD

**Issue by 1496nuonuo** [DeepanshuKarnwal/AD#1]
Hi! You do very meaningful work in Oran AD. I have read many releases of the Oran AD Xapps. What confuse me most are the criterion of the anomalous labels. What does it mean? How can we judge it an anomaly datapoint or not?

**Comment by DeepanshuKarnwal** [comment]
Thank you. In AD xApp, if there is sudden drop in throughput or radio parameters (KPIs) of any user, those samples are labelled as anomalous otherwise normal.

**Comment by 1496nuonuo** [comment]
So are the labels just given by observation? Or are there other more concrete formulas?

**Comment by 1496nuonuo** [comment]
I think the mathematical formulations or some quantization of the drop in the throughput or radio parameters in a time series are a little non-trivial.

**Comment by DeepanshuKarnwal** [comment]
You are right! Sudden drops (not gradual) of 70% or more in throughput kpi or similar loss in radio parameters qualify for a sample to be anomalous

**Comment by DeepanshuKarnwal** [comment]
You are right! But we expect for a user to have smooth QoE i.e throughput to be maintained (normal behavior). Sudden drops (not gradual) of 70% or more in throughput kpi or similar loss in radio parameters qualify for a sample to be anomalous.

**Comment by 1496nuonuo** [comment]
Hi! I find target throughput in the metrics collection ''[ue.csv](https://github.com/o-ran-sc/ric-app-ad/tree/master/src)'' in the Oran-sc ad xapps. Would you tell me how to compute the target throughput in this collection? Thanks!
